### PR TITLE
[INLONG-3652][Agent] Improve TestBinlogOffsetManager unit test

### LINKED ISSUE #3652: [Improve][Agent] Improve TestBinlogOffsetManager unit test
### Description

image

### InLong Component

InLong Agent

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by pocozh**
### Title Name: [INLONG-3652][Agent] Improve TestBinlogOffsetManager unit test

where *XYZ* should be replaced by the actual issue number.

Fixes #3652 

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve?*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

*(Please pick either of the following options)*

- [ ] This change is a trivial rework/code cleanup without any test coverage.

- [ ] This change is already covered by existing tests, such as:
  *(please describe tests)*

- [ ] This change added tests and can be verified as follows:

  *(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a follow-up issue for adding the documentation

### CODE REVIEW

**Review by healchow — APPROVED** [review]

File: inlong-agent/agent-plugins/src/test/java/org/apache/inlong/agent/plugin/sources/TestBinlogOffsetManager.java [hunk 1]
```diff
@@ -28,16 +27,22 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
 public class TestBinlogOffsetManager {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(TestBinlogOffsetManager.class);
```
Comment (healchow) [inline inlong-agent/agent-plugins/src/test/java/org/apache/inlong/agent/plugin/sources/TestBinlogOffsetManager.java:35]: Remove this unused line.

File: inlong-agent/agent-plugins/src/test/java/org/apache/inlong/agent/plugin/sources/TestBinlogOffsetManager.java [hunk 2]
```diff
@@ -28,16 +27,22 @@
...
     private static Path testDir;
```
Comment (healchow) [inline inlong-agent/agent-plugins/src/test/java/org/apache/inlong/agent/plugin/sources/TestBinlogOffsetManager.java:36]: Convert to local.

File: inlong-agent/agent-plugins/src/test/java/org/apache/inlong/agent/plugin/sources/TestBinlogOffsetManager.java [hunk 3]
```diff
@@ -28,16 +27,22 @@
...
     private static AgentBaseTestsHelper helper;
+    private static String fileName = "test.txt";
```
Comment (healchow) [inline inlong-agent/agent-plugins/src/test/java/org/apache/inlong/agent/plugin/sources/TestBinlogOffsetManager.java:38]: Add `final`.

File: inlong-agent/agent-plugins/src/test/java/org/apache/inlong/agent/plugin/sources/TestBinlogOffsetManager.java (hunk 2 above)
Comment (pocozh) [reply inlong-agent/agent-plugins/src/test/java/org/apache/inlong/agent/plugin/sources/TestBinlogOffsetManager.java:36]: > Convert to local.

done

File: inlong-agent/agent-plugins/src/test/java/org/apache/inlong/agent/plugin/sources/TestBinlogOffsetManager.java (hunk 3 above)
Comment (pocozh) [reply inlong-agent/agent-plugins/src/test/java/org/apache/inlong/agent/plugin/sources/TestBinlogOffsetManager.java:38]: > Add `final`.

done

File: inlong-agent/agent-plugins/src/test/java/org/apache/inlong/agent/plugin/sources/TestBinlogOffsetManager.java (hunk 1 above)
Comment (pocozh) [reply inlong-agent/agent-plugins/src/test/java/org/apache/inlong/agent/plugin/sources/TestBinlogOffsetManager.java]: > Remove this unused line.

done

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**